src/metabase/models/model_index.clj

```diff
@@ -187,6 +187,7 @@
 
 (search/define-spec "indexed-entity"
   {:model        :model/ModelIndexValue
+   :visibility   :app-user
```

Comment:
Not sure what to call this 

- regular user?
- actual user?
- non-impersonated-or-sandbox-user?

Reply:
lol this is a hard one, maybe @noahmoss can help here :D